recipe/meta.yaml

```diff
@@ -22,6 +22,8 @@ build:
     - libwebp  # [win or (osx and arm64)]
```

Comment:
build number should become 
number: 1

Reply:
Another victim of the merge mess I created.  Just bumped it.